pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -270,6 +278,35 @@ func evictPods(inputPods []*v1.Pod,
 	}
 }
 
+func podToleratesTaints(pod *v1.Pod, taintsOfNodes [][]v1.Taint) bool {
+	canBeEvicted := true
+taintNodeLoop:
+	for _, taintsForNode := range taintsOfNodes {
+		if len(pod.Spec.Tolerations) < len(taintsForNode) {
```

Comment:
Is `taintsForNode` guaranteed to be always duplicate items free?

Reply:
The k8s api rejects taints with the same key and same effect

So technically you can have a slice of `Taint` with same key but different effect values. Which means a pod needs to either tolerate this key with `Exists` value or have two tolerations with same key but different effect values i.e `NoSchedule` and `NoExecute`